employee-onboarding-A2A-reference-implementation/src/auth/asgardeo.py

```
@@ -0,0 +1,649 @@
+    - Token Exchange (RFC 8693) for delegation and downscoping
+    """
+    
+    def __init__(self):
+        self.settings = get_settings()
+        self._actor_token: Optional[ActorToken] = None
+    
+    def _create_fresh_client(self) -> httpx.AsyncClient:
+        """Create a fresh HTTP client with no cookies (for each auth flow)."""
+        return httpx.AsyncClient(
+            timeout=30.0, 
+            verify=False,
```

Ruff (0.15.2)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)